pkgs/moduleit/entrypoint.nix

```diff
@@ -1,7 +1,14 @@
 { pkgs ? import <nixpkgs> { }
 , configPath
 }:
-
+let pruneVersion = version:
+  let
+    parts = pkgs.lib.strings.splitString "." version;
+    major = builtins.elemAt parts 0;
+    minor = builtins.elemAt parts 1;
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/lib/versions.nix there are already major and minor functions you could use here.

`pruneVersion = lib.majorMinor`

Reply:
:+1: I thought there might be.